packages/web3-shared/evm/constants/index.ts

```diff
@@ -25,9 +25,13 @@ import OpenseaAPI from '@masknet/web3-constants/evm/opensea-api.json'
 import CryptoArtAI from '@masknet/web3-constants/evm/cryptoartai.json'
 import { hookTransform, transform, transformFromJSON } from './utils'
 
-function getEnvConstants(key: string) {
+function getEnvConstants(key: 'WEB3_CONSTANTS_RPC' | 'WEB3_CONSTANTS_EXPLORER') {
     try {
-        return process.env[key] ?? ''
+        const map = {
```

Comment:
move to top level?

Reply:
need i provide eslint rule? "enforce static process.env expression"

Reply:
Keep this in the try statement cause it will throw an error if no env is given.